app/policies/user_policy.rb

```diff
@@ -1,27 +1,34 @@
 class UserPolicy < ApplicationPolicy
-  USER_MANAGEMENT_ROLES = %w(superadmin policy_and_abuse open_doors support tag_wrangling).freeze
-  POLICY_ROLES = %w(superadmin policy_and_abuse).freeze
+  # Roles that allow:
+  # - troubleshooting for a user
+  # - managing invitations
+  # - updating a user's email and roles (e.g. wranglers, archivists, not admin roles)
+  TROUBLESHOOT_ROLES = %w(superadmin policy_and_abuse open_doors support tag_wrangling).freeze
```

Comment:
idk, do we want to call this just `MANAGE_ROLES` and `can_manage_users`? I get what it's going for and didn't really bat an eye at it, but when I got down to the `<% if policy(User).can_troubleshoot_users? %>` surrounding  the "Manage Users" menu, I was like, "Ehhh, that's actually a bit confusing. It seems like that should just be on the Troubleshoot buttons."